src/locales/en.ts

```diff
@@ -214,5 +237,23 @@ export const en = {
     create: 'Create a new app',
     getStarted: ' to get started',
     deleteDialog: (name: string) => `Are you sure you want to delete ${name}? This action cannot be undone.`
+  },
+  router: {
+    portalTitle: '| Developer Portal',
```

Comment:
Should this formatting happen in the i18n part?

Reply:
Not the pipe, I'll refactor.